src/components/Dashboard.js

```diff
@@ -0,0 +1,92 @@
+import React,{useState,useEffect} from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import CreateModal from './CreateModal';
+import Cards from './Cards';
+import Modal  from './Modal';
+import './Dashboard.css';
+
+function Dashboard(){
+
+    const categories = ["Dev-Tasks","Updates-Push","Updates-Pull","Prod-Issue","Code-Review","Meetings"];
+    const priorities = ["P0","P1","P2","P3","P4"];
+    let [cardsData,setCards] = useState(getLocalCards());
+    let [modalVisible,setModal] = useState(false);
+    let [content,setContent] = useState('');
+    let [category,setCategory] = useState('Dev-Tasks');
+    let [priority,setPriority] = useState('P0');
+    let [date,setDate] = useState(new Date())
+    let [modalUpdate, setModalUpdate] = useState(false);
+    let [id,setId] = useState('');
+    function toggleModal(){
+        setModal(!modalVisible);
+    }
+    function closeModal(){
+        setModalUpdate(false);
+        setModal(false);
+        setContent('');
+        setCategory('');
+        setPriority('');
+        setDate('');
+    }
+    
+
+    function getLocalCards(){
```

Comment:
this function can be stored in a helper file, no need make it a part of this component.